packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/FirebaseAdMobPlugin.java

```diff
@@ -93,6 +93,13 @@ private void callShowAd(int id, MethodCall call, Result result) {
       result.error("ad_not_loaded", "show failed, the specified ad was not loaded id=" + id, null);
       return;
     }
+    if (call.argument("anchorOffset") != null) {
+      ad.anchorOffset = ((Integer) call.argument("anchorOffset")).intValue();
+    }
+    if (call.argument("anchorType") != null) {
+      ad.anchorType = ((Integer) call.argument("anchorType")).intValue();
```

Comment:
We may want to use Strings here. It is all to easy to misinterpret an integer across three code files.